scripts/dailyupdate

```diff
@@ -7,6 +7,10 @@ source ~/parlparse/scripts/consts
 # Update register of members interests
 cd ~/parlparse
 poetry run python -m pyscraper.regmem download-all-registers --chamber commons --quiet
+poetry run python -m pyscraper.regmem download-latest-register --chamber scotland --quiet
```

Comment:
why is the commons fetching all registers and the rest only fetching the latest?

Reply:
No strong reason - as it runs daily they should all download-latest-register